database/migrations/2020_12_26_154040_create_recat_categories_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRecatCategoriesTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('recat_categories', function (Blueprint $table) {
+            $table->id();
+            $table->string('code')->comment('The code for the category')->unique();
```

Comment:
Does unique also create an index? Might be worth indexing if not for performance

Reply:
Yes it does on InnoDB